chapter_recurrent-neural-networks/rnn-concise.md

```diff
@@ -214,14 +265,31 @@ model = model.to(device)
 d2l.predict_ch8('time traveller', 10, model, vocab, device)
 ```
 
+```{.python .input}
+#@tab tensorflow
+device_name = d2l.try_gpu()._device_name
+strategy = tf.distribute.OneDeviceStrategy(device_name)
+with strategy.scope():
+    model = RNNModel(rnn_layer, vocab_size=len(vocab))
+params = d2l.get_params(len(vocab), num_hiddens)
+predict_ch8('time traveller ', 10, model, vocab, params)
```

Comment:
`'time traveller '` -> `'time traveller'`
Besides, why don't we save `predict_ch8` in d2l as in other frameworks?

Reply:
Thanks. That is a typo. Fixed now.